lusidtools/cocoon/transaction_type_upload.py

```diff
@@ -58,3 +58,91 @@ def create_transaction_type_configuration(api_factory, alias, movements):
     )
 
     return response
+
+
+def replace_current_transaction_alias(
+    api_factory, updated_alias, movement, properties=None
```

Comment:
"movement" here "movements" in your docstring

Reply:
Thanks. Updated to "movements"